plot/plotutils.py

```diff
@@ -87,5 +86,4 @@ def plot_chainconsumer(chains, output_dir, infer_type, true_param_mean=[]):
     for ax in ax_list:
         ax.grid(False)
 
-    plt.savefig(
-        os.path.join(output_dir, f'plot_{infer_type}_cc_{timestamp()}.png'))
+    plt.savefig(os.path.join(output_dir, f'{infer_type}_cc.png'))
```

Comment:
Quick request - can you have all figures saved as both png and pdf?  pdfs are what we'll put into the paper draft, and pngs are easier to thumbnail scroll through

Reply:
added pdf saving as well